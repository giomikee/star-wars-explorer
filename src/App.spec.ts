import { shallowMount } from '@vue/test-utils'
import { describe, expect, it } from 'vitest'
import App from './App.vue'

const createWrapper = () => shallowMount(App)

describe('Given App component', () => {
  describe('When component is rendered', () => {
    it('Then it should match component snapshot', () => {
      const { element } = createWrapper()

      expect(element).toMatchSnapshot()
    })
  })
})
