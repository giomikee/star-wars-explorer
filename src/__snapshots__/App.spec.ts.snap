// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Given App component > When component is rendered > Then it should match component snapshot 1`] = `
<div
  class="v-application v-theme--dark v-layout v-layout--full-height v-locale--is-ltr"
>
  <div
    class="v-application__wrap"
  >
    
    <router-view />
    
  </div>
</div>
`;
